tests/integration/test_correlation.py

```diff
@@ -100,6 +101,7 @@ async def test_correlation_id_isolation():
         ("BAD_ID", InvalidCorrelationIdError),
         ("", InvalidCorrelationIdError),
         (str(VALID_CORRELATION_ID), InvalidCorrelationIdError),
+        (NON_V4_UUID, InvalidCorrelationIdError),
```

Comment:
why does this parameter list have the exception type when it's always the same anyway?

Reply:
also, the test checks the return value, but the function does not return anything, this is misleading